packages/ai-core/src/browser/ai-activation-service.ts

```diff
@@ -33,23 +33,23 @@ export class AIActivationService implements FrontendApplicationContribution {
     @inject(PreferenceService)
     protected preferenceService: PreferenceService;
 
-    protected isExperimentalEnabledKey: ContextKey<boolean>;
+    protected isAIEnabledKey: ContextKey<boolean>;
```

Comment:
nit: in general I personally prefer camel-casing like `isAiEnabledKey` with abbreviations like AI. But we don't use that consistently, so I'm fine with this as well.

Reply:
Yeah, I tend towards 'AI' as it is an abbreviation

Reply:
renamed!